package.json

```diff
@@ -1,18 +1,21 @@
 {
-  "name": "switchboard-api-lambda",
+  "name": "switchboardApiLambda",
   "projectName": "cms-fronts::switchboard-api-lambda",
   "description": "Lambda to handle switchboard API requests",
   "version": "1.0.0",
   "private": true,
   "license": "Apache-2.0",
+  "cloudformation": false,
+  "buildDir": "tmp/lambda",
+  "isAwsLambda": true,
+  "riffraffFile": "./riff-raff.yaml",
```

Comment:
I think this is the default anyway for `node-riffraff-artefact`

Reply:
I thought so too but I've seen in the other lambdas that it doesn't find the file in the root directory unless this gets specified.

Reply:
oh super, yeah, never mind then...

Probably because [that's not right](https://github.com/guardian/node-riffraff-artefact/blob/master/src/settings.js#L23). I'll have a look and see if I can update, have some other bits I wanted to change in the plugin anyway.